src/contracts/pods/DelayedWithdrawalRouter.sol

```diff
@@ -56,7 +56,8 @@ contract DelayedWithdrawalRouter is Initializable, OwnableUpgradeable, Reentranc
      * @notice Creates a delayed withdrawal for `msg.value` to the `recipient`.
      * @dev Only callable by the `podOwner`'s EigenPod contract.
      */
-    function createDelayedWithdrawal(address podOwner, address recipient) external payable onlyEigenPod(podOwner) {
+    function createDelayedWithdrawal(address podOwner, address recipient) external payable onlyEigenPod(podOwner) onlyWhenNotPaused(PAUSED_DELAYED_WITHDRAWAL_CLAIMS) {
+        require(recipient != address(0), "DelayedWithdrawalRouter.createDelayedWithdrawal: recipient cannot be zero address");
```

Comment:
can we add unit test coverage for these changes?

Reply:
added: [097c530](https://github.com/Layr-Labs/eigenlayer-contracts/pull/30/commits/097c530a90d7eac45accb84c575c951fa4b155e5)